content/en/docs/ops/configuration/extensibility/wasm-module-distribution/index.md

```diff
@@ -49,13 +49,14 @@ spec:
          type_urls: [ "type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm"]
 {{< /text >}}
 
-The second `EnvoyFilter` resource provides configuration for the filter, which is composed as an `EXTENSION_CONFIG` patch and will be distributed to the proxy as an Envoy [`Extension Configuration`](https://www.envoyproxy.io/docs/envoy/latest/configuration/overview/extension) (ECDS) resource.
-Once this update reaches Istio agent, it will download the Wasm module file and store it at the local file system.
-If the download fails, Istio-agent will reject the `Extension Configuration` update and prevent bad Wasm filter configuration from reaching Envoy.
-Most of this `EnvoyFilter` configuration is boilerplate. The important parts are:
+The second `EnvoyFilter` provides configuration for the filter, which is an `EXTENSION_CONFIG` patch and will be distributed to the proxy as an Envoy [`Extension Configuration`](https://www.envoyproxy.io/docs/envoy/latest/configuration/overview/extension) (ECDS) resource.
+Once this update reaches the Istio agent, the agent will download the Wasm module and store it in the local file system.
+If the download fails, the agent will reject the `Extension Configuration` update to protect invalid Wasm filter configuration from reaching the Envoy proxy.
+Because of this protection, with the initial fetch timeout being set to 0, the listener update will not become effective and invalid Wasm filter will not disturb the traffic.
```

Comment:
```suggestion
Because of this protection, with the initial fetch timeout being set to 0, the listener update will not become effective and an invalid Wasm filter will not disturb traffic.
```